### PR TITLE
Update twig templates override in modules

### PR DESCRIPTION
**Pull request by atomiix**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Update the twig template override in modules to reflect the changes in PrestaShop 8.0. https://github.com/PrestaShop/PrestaShop/pull/27044 needs to be merged first.
| Fixed ticket? |

### CODE REVIEW

**Review by kpodemski — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]
LGTM 👍 

I guess we should wait for your other PR to be merged before merging this one

**Review by PierreRambaud — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
thank you @atomiix  !

**Comment by matks** [comment]
@atomiix can you write the documentation for this feature? 😉

**Pull request merged**
